idea-plugin/src/test/java/com/palantir/gradle/versions/intellij/VersionPropsCodeInsightTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.gradle.versions.intellij;
+
+import com.intellij.codeInsight.completion.CompletionType;
+import com.intellij.testFramework.fixtures.LightJavaCodeInsightFixtureTestCase;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class VersionPropsCodeInsightTest extends LightJavaCodeInsightFixtureTestCase {
```

Comment:
Nice! Actual tests.

Reply:
Quite pleased with these, quite basic at the moment but can expand coverage in the future. Currently I am using real packages in maven central for the tests - is this a bad idea? would I be better of mocking this so I'm not relying on maven and the packages not changing